examples/ad_gravity.py

```diff
@@ -0,0 +1,44 @@
+import taichi as ti
+
```

Comment:
Just wonder: Dont we need to keep `ti.init` here anymore?

Reply:
```suggestion
ti.init()

```
Yes, sometimes Taichi developments can break these examples without `ti.init()`.